controllers/netbot.js

```diff
@@ -2733,29 +2806,13 @@ class netBot extends ControllerBot {
         break;
       case "policy:delete":
         (async () => {
-          const policyIDs = value.policyIDs;
```

Comment:
will this break existing clients?

Reply:
I added this a few weeks ago, not used yet.